scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -797,6 +797,9 @@ task FilterByOrientationBias {
     File pre_adapter_metrics
     Array[String]? artifact_modes
 
+    # TODO: Do not pass artifact_modes as an empty array [].  https://github.com/broadinstitute/gatk/issues/5025
```

Comment:
Could you solve this via (I didn't test this syntax)
```
Boolean run_ob_filter = select_first([run_orientation_bias_filter, true]) && length(final_artifact_modes) > 0
```

Reply:
That would solve it in the workflow.  Not necessarily the task if used in another workflow.

Done.